src/visualization/types/Table/view.tsx

```diff
@@ -81,71 +89,145 @@ const TableGraphs: FC<Props> = ({properties, result}) => {
-                        <span className="value">{v}</span>
-                      </Fragment>
-                    )
-                  })
-                return (
-                  <div
-                    key={table.id}
-                    className={classer}
-                    onClick={() => updateSelectedTable(table.name)}
-                  >
-                    {display}
-                  </div>
-                )
-              })}
-            </div>
-          </DapperScrollbars>
-        </div>
-      )}
-      {!!_selectedTable && (
-        <Table
-          key={_selectedTable?.name}
-          table={_selectedTable}
-          sort={sortOptions}
-          updateSort={updateSortOptions}
-          properties={properties}
-        />
-      )}
-      {(!_selectedTable || !_selectedTable.data.length) && (
-        <EmptyGraphMessage message="This table has no data" />
-      )}
-    </div>
-  )
+  if (isFlagEnabled('useGiraffeGraphs')) {
+    const parsed = parseFromFluxResults(result)
+    const fluxResponse = parsed.tableData.join('\n')
+    const config: Config = {
+      fluxResponse,
+      layers: [
+        {
+          type: 'table',
+          properties: {
+            colors: DEFAULT_TABLE_COLORS,
+            tableOptions: {
+              fixFirstColumn: false,
+              verticalTimeAxis: true,
+            },
+            fieldOptions: [
+              {
+                displayName: '_start',
+                internalName: '_start',
+                visible: true,
+              },
+              {
+                displayName: '_stop',
+                internalName: '_stop',
+                visible: true,
+              },
+              {
+                displayName: '_time',
+                internalName: '_time',
+                visible: true,
+              },
+              {
+                displayName: '_value',
+                internalName: '_value',
+                visible: true,
+              },
+              {
+                displayName: '_field',
+                internalName: '_field',
+                visible: true,
+              },
+              {
+                displayName: '_measurement',
+                internalName: '_measurement',
+                visible: true,
+              },
+              {
+                displayName: 'cpu',
+                internalName: 'cpu',
+                visible: true,
+              },
+              {
+                displayName: 'host',
+                internalName: 'host',
+                visible: true,
+              },
+            ],
+            timeFormat: properties.timeFormat,
+            decimalPlaces: {
+              digits: 3,
+              isEnforced: true,
+            },
+          },
+          timeZone: 'Local',
+          tableTheme: theme,
+        },
+      ],
+    }
+    return (
+      <HoverTimeProvider>
+        <Plot config={config} />
+      </HoverTimeProvider>
+    )
+  } else {
```

Comment:
```suggestion
```

we're returning in the `if` statement; ditching the `else` will leave this component's default/happy path as the main pathway